openssl/openssl.php

```diff
@@ -418,7 +428,8 @@ function openssl_pkcs12_export(
     &$output,
     #[LanguageLevelTypeAware(['8.0' => 'OpenSSLAsymmetricKey|OpenSSLCertificate|array|string'], default: 'resource|array|string')] $private_key,
     string $passphrase,
-    array $options
+    #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $args,
+    #[PhpStormStubsElementAvailable(from: '7.1')] array $options = []
```

Comment:
optional `$options` is called `$args`. Probably we can rename it to `$arguments` to avoid duplication

Reply:
We can't rename `options` because `options` is the name that comes from reflection and if we will change it then it will break named arguments. If you mean to rename `args` then it actually no matter what this name is as it's used in PHP < 8 and it will not bring problems with named arguments